addon/utils/_private/constants.ts

```diff
@@ -3,6 +3,8 @@
  * we've added a, del, ins to the list since we assume they only contain phrasing content in the editor
  * we've removed br from the list to be inline with editor behaviour, which treats it as a block
  **/
+
+export const HEADING_ELEMENTS = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'] as const;
```

Comment:
This takes the JSdoc from `PHRASING_CONTENT`, we should position it differently.

Reply:
Done